recorder.go

```diff
@@ -178,8 +190,8 @@ func NewRecorder(opts Options) basictracer.SpanRecorder {
 	if _, found := opts.Tags[ComponentNameKey]; !found {
 		opts.Tags[ComponentNameKey] = path.Base(os.Args[0])
 	}
-	if _, found := opts.Tags[GUIDKey]; !found {
-		opts.Tags[GUIDKey] = genSeededGUID()
+	if _, found := opts.Tags[GUIDKey]; found {
+		panic(fmt.Sprintf("Passing in your own %v is no longer supported", GUIDKey))
```

Comment:
Sorry, I missed that.
ᐧ

On Tue, Sep 6, 2016 at 3:38 PM, spoons <notifications@github.com> wrote:

> In recorder.go
> <https://github.com/lightstep/lightstep-tracer-go/pull/32#discussion_r77731770>
> :
>
> > @@ -178,8 +190,8 @@ func NewRecorder(opts Options) basictracer.SpanRecorder {
> >  	if _, found := opts.Tags[ComponentNameKey]; !found {
> >  		opts.Tags[ComponentNameKey] = path.Base(os.Args[0])
> >  	}
> > -	if _, found := opts.Tags[GUIDKey]; !found {
> > -		opts.Tags[GUIDKey] = genSeededGUID()
> > +	if _, found := opts.Tags[GUIDKey]; found {
> > +		panic(fmt.Sprintf("Passing in your own %v is no longer supported", GUIDKey))
>
> Sorry if I missed a conversation, but my preference is to avoid all panics
> in this library. Maybe just maybeLogError?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/lightstep/lightstep-tracer-go/pull/32/files/96f617d518f82bb9a6344d9684dcc38f4b097dd9#r77731770>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADdiiTog1hS-3MNzCq2o4Qti8SSLxs1Lks5qnetugaJpZM4J2H3a>
> .
>